data/net/minecraft/commands/arguments/coordinates/BlockPosArgument.mapping

```diff
@@ -2,8 +2,12 @@ CLASS net/minecraft/commands/arguments/coordinates/BlockPosArgument
 	METHOD getLoadedBlockPos (Lcom/mojang/brigadier/context/CommandContext;Ljava/lang/String;)Lnet/minecraft/core/BlockPos;
 		ARG 0 context
 		ARG 1 name
+	METHOD getOrLoadBlockPos (Lcom/mojang/brigadier/context/CommandContext;Ljava/lang/String;)Lnet/minecraft/core/BlockPos;
+		ARG 0 context
+		ARG 1 argumentName
```

Comment:
For consistency, this parameter should be named `name`. (It is understood that the `static` methods in argument classes retrieve arguments from the given command context.)
```suggestion
		ARG 1 name
```